src/components/Nav.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CityFilter from './City/CityFilter';
+import './styles/Nav.css';
+
+const Nav = ({ onFilterChange }) => (
+  <header>
+    <nav className="flex">
+      <h1>AirTrakr</h1>
+      <CityFilter onFilterChange={onFilterChange} />
+    </nav>
+  </header>
```

Comment:
- [ Required ] Good job so far, however, I noticed that your design of the App differs slightly from the one on `Behance.net`. The design on Behance.net has a top bar section that has the following items; a `title`, `microphone` icon, and a `settings` icon. Please ensure that your design matches this part of the Behance design.

## Behance design
![Screenshot from 2023-08-11 15-04-48](https://github.com/DevaWinner/airtrackr/assets/16392046/4b9af51c-3608-4f5b-8779-67d6d419704c)

## Your design
| Home page | Details page |
|-----------------|-------------------|
|![Screenshot from 2023-08-11 15-10-00](https://github.com/DevaWinner/airtrackr/assets/16392046/ea5550bd-7684-48f1-a9d2-dda546528044)|![Screenshot from 2023-08-11 15-10-28](https://github.com/DevaWinner/airtrackr/assets/16392046/e170805a-3706-40c8-9be7-c825809db2ce)|

Reply:
Thank you for the review. But may I add that the said components don't add any use to my project. I need that space for my own filter element which is the search input field.

The microphone and setting are used for filtering the data on the design app. Mine is an input field, so I need to place it somewhere up there to meet the design. And so, I will be crowding the navbar with components that aren't useful and won't find sufficient space for the input element that I need to filter my data.

The `town/city` on the details page is the same with my own city name element.

Reply:
Hi @DevaWinner 

Thanks for reaching out, however, following the design guidelines is very important, imagine a situation where a client gives you a design document but as a developer you choose not to follow certain aspects of the design document. The client may not be very happy with the final project even if it's functioning properly. The items I have pointed out right now should not be functional at this point, but in the future, you could work on the `settings` button so that a user is able to customize the look and feel of the App, or you could work on the `microphone` icon to enable a user to speak into your App if they want to filter their city of choice.

Reply:
I hope you don't mind, but where will you suggest I place my search field and still follow the design. I am asking because I have my other peers also place their filter components there and it was approved.

I am just contemplating where else I will place my search box if I replace it with a settings and mic icon. The mobile design doesn't give much space for too many items. 

Can you please assist me on this?

Reply:
Hi @DevaWinner,

I think you could place the search bar just below the `nav` menu.

```JavaScript
import React from 'react';
import PropTypes from 'prop-types';
import CityFilter from './City/CityFilter';
import './styles/Nav.css';

const Nav = ({ onFilterChange }) => (
  <header>
    <nav className="flex">
      <h1>AirTrakr</h1>
    </nav>
    <CityFilter onFilterChange={onFilterChange} />
  </header>
);

Nav.propTypes = {
  onFilterChange: PropTypes.func.isRequired,
};

export default Nav;
```

Reply:
Thank you for the suggestion. I have implemented it.